com.unity.netcode.gameobjects/Tests/Runtime/Metrics/PacketMetricsTests.cs

```diff
@@ -0,0 +1,66 @@
+#if MULTIPLAYER_TOOLS
+using System;
+using System.Collections;
+using System.Linq;
+using NUnit.Framework;
+using Unity.Collections;
+using Unity.Multiplayer.Tools.MetricTypes;
+using Unity.Multiplayer.Tools.NetStats;
+using Unity.Netcode.RuntimeTests.Metrics.Utility;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Netcode.RuntimeTests.Metrics
+{
+    internal class PacketMetricsTests
+    {
+        [UnityTest]
+        public IEnumerator TrackPacketSentMetric()
+        {
+            const int clientCount = 1;
+            MultiInstanceHelpers.Create(clientCount, out var server, out var clients, 60, MultiInstanceHelpers.InstanceTransport.UTP);
+
+            server.StartServer();
+            clients[0].StartClient();
+
+            var serverMetrics = (NetworkMetrics)server.NetworkMetrics;
+            var waitForMetricValues = new WaitForCounterMetricValue(serverMetrics.Dispatcher, NetworkMetricTypes.PacketSent, metric => metric > 0);
+
+            using (var writer = new FastBufferWriter(sizeof(uint), Allocator.Temp))
```

Comment:
nit, might be worth turning this into a utility function since its implementation isnt relevant to the tests

Reply:
We already use this format over all tests. I'm not sure how much more I can turn this into a utility function since depending on the test scenario, these 3 values can change.